pkgs/applications/editors/rstudio/remove-quarto-from-generator.patch

```diff
@@ -0,0 +1,29 @@
+--- a/src/cpp/session/CMakeLists.txt
++++ b/src/cpp/session/CMakeLists.txt
+@@ -45,8 +45,6 @@
+ 
+    if(EXISTS "${RSTUDIO_TOOLS_ROOT}/quarto")
+       set(RSTUDIO_DEPENDENCIES_QUARTO_DIR "${RSTUDIO_TOOLS_ROOT}/quarto")
```

Comment:
So does the "then"-branch always apply? I'm confused what the variable `RSTUDIO_DEPENDENCIES_QUARTO_DIR` is now actually set to, and how the build will find `quarto`.

Reply:
Ok, but then you could remove the whole `if` construct, right? Maybe that would be clearer.

Reply:
So as you suggested, the pandoc patch and quarto patch is separated. Also the full quarto statement is removed. See commit https://github.com/NixOS/nixpkgs/pull/185154/commits/bb9fcc09880a1f70d0c23f24483b7baa9d8119ac.